logstash-core/lib/logstash/pipeline.rb

```diff
@@ -159,7 +160,8 @@ def run
     shutdown_flusher
     shutdown_workers
 
-    @logger.log("Pipeline #{@pipeline_id} has been shutdown")
```

Comment:
As a logstash user, I would appreciate to have this log at `info` level for example.